freezer.js

```diff
@@ -12,39 +12,48 @@ function addItemToFreezer() {
     id: Math.random()
   });
   freezerContents.sort();
-  document.getElementById('item_input').value='';
+  document.getElementById('item_input').value = '';
+}
+
+function createDivs(numberOfDivsToCreate) {
+  const divs = [];
+
+  for (var i = 0; i < numberOfDivsToCreate; i++) {
+    divs.push(document.createElement('div'))
+  };
+
+  return divs;
+}
+
+function appendElementsToElement(elementToAppendTo, elementsToAppend) {
+  for (var i = 0; i < elementsToAppend.length; i++) {
+    elementToAppendTo.appendChild(elementsToAppend[i]);
+  }
 }
 
 function createList() {
   //create the list element
   list = document.createElement('div');
   for (var i = 0; i < freezerContents.length; i++) {
-    //create the list item
-    var listNumber = document.createElement('div');
-    var listItem = document.createElement('div');
-    var row = document.createElement('div');
-    var minusButton = document.createElement('div');
-    var plusButton = document.createElement('div');
+    //create the list items
+    var [listNumber, listItem, row, minusButton, plusButton] = createDivs(5);
```

Comment:
This is a JavaScript feature called List Restructuring (or simply  restructuring).

Given an array of 5 elements `[div,div,div,div,div]`, I can declare a variable `var [one, two, three, four, five]` which  allows assigns each of the names to each of the elements within the array, so the first `div` is `one`,  the second is  `two` so on so forth.

Putting it together gives us:

`var [one, two, three, four, five] = [div,div,div,div,div];`